.gitmodules

```diff
@@ -19,3 +16,6 @@
 [submodule "sw/ext/mavlink"]
 	path = sw/ext/mavlink
 	url = https://github.com/paparazzi/mavlink.git
+[submodule "sw/ext/chibios"]
+	path = sw/ext/chibios
+	url = git@github.com:ChibiOS/ChibiOS.git
```

Comment:
Please use https URLs for the submodules, so it works without ssh keys (as on the CI server) and through a proxy